django/views/i18n.py

```diff
@@ -17,6 +17,7 @@
     LANGUAGE_SESSION_KEY, check_for_language, get_language, to_locale,
 )
 from django.utils.translation.trans_real import DjangoTranslation
+from django.utils.six.moves.urllib.parse import unquote
```

Comment:
You'll have to move this import before the `django.utils.translation` one to appease isort.